src/core/sys/darwin/mach/loader.d

```diff
@@ -159,252 +581,2853 @@ version (CoreDdoc)
         uint filesize;
 
         /// Maximum VM protection.
-        int maxprot;
+        vm_prot_t maxprot;
 
         /// Initial VM protection.
-        int initprot;
+        vm_prot_t initprot;
 
-        /// Number of sections in this segment.
+        /// Number of sections in segment.
         uint nsects;
 
         /// Flags.
         uint flags;
     }
 
-    /// Represents a segment command in a Mach-O file for 64-bit architecture.
+    /*
+     * The 64-bit segment load command indicates that a part of this file is to
+     * be mapped into a 64-bit task's address space. If the 64-bit segment has
+     * sections then section_64 structures directly follow the 64-bit segment
+     * command and their size is reflected in cmdsize.
+     */
     struct segment_command_64
     {
-        /// Type of load command, i.e. `LC_SEGMENT`.
+        /// LC_SEGMENT_64.
         uint cmd;
 
-        /// The size of this segment, includes size of section structs.
+        /// Includes sizeof section_64 structs.
         uint cmdsize;
 
-        /// The name of this segment.
-        char[16] segname = 0;
+        /// Segment name.
+        char[16] segname;
```

Comment:
Ditto throughout

Reply:
Is it only necessary for static arrays of type `char`?

Reply:
Fixed.